playbook-website/app/javascript/components/GlobalPropsAndTokens/ExamplesPage/Examples/DisplayGlobalProps.tsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { Flex, Card, Body, Caption } from "playbook-ui";
+import ShowPage from "../../Templates/ShowPage";
+import PropsExamplesTable from "../../Templates/Subcomponents/PropsExamplesTable";
+import ExampleCodeCard from "../../Templates/Subcomponents/ExampleCodeCard";
+import * as DisplayImages from './DisplayImages'
+const DisplayGlobalProps = () => {
+
+  console.log(DisplayImages)
+
+  type DisplayKey = keyof typeof DisplayImages;
+
+  const VisualGuideExample = ({ display }: { display: DisplayKey }) => {
+    return (
+      <>
+        <Flex orientation="column" alignItems="center" gap="xs">
+          <Card
+            className="visual_guide_card_line_height"
+            display="flex"
+            alignItems="center"
+            justifyContent="center"
+          >
+            <img
+              alt={`Display example of ${display}`}
+              src={DisplayImages[display]}
+            />
+          </Card>
+          <Body text={display} />
+        </Flex>
+      </>
+    )
+  }
+
+  const DisplayCard = () => {
+    return (
+      <Flex gap="sm" wrap>
+        <VisualGuideExample display="block" />
+        <VisualGuideExample display="inline" />
+        <VisualGuideExample display="inlineBlock" />
+        <VisualGuideExample display="none" />
+        <VisualGuideExample display="flex" />
+        <VisualGuideExample display="inlineFlex" />
+        <VisualGuideExample display="grid" />
+      </Flex>
+    );
+  };
+
+  const values = [
+    "block",
+    "inline",
+    "inlineBlock",
+    "none",
+    "flex",
+    "inlineFlex",
+    "grid",
+  ];
+
+const sizes = {
+    xs: { label: "xs", minWidth: "", maxWidth: "575px", descrption: "Extra small screens" },
+    sm: { label: "sm", minWidth: "576px", maxWidth: "767px", descrption: "Small screens" },
+    md: { label: "md", minWidth: "768px", maxWidth: "991px", descrption: "Medium screens" },
+    lg: { label: "lg", minWidth: "992px", maxWidth: "1199px", descrption: "Large screens" },
+    xl: { label: "xl", minWidth: "1200px", maxWidth: "", descrption: "Extra large screens" },
+    default: { label: "default" , minWidth: "", maxWidth: "", descrption: "Default is used as the final fallback and is not tied to a specific screen szie." },
+};
+
+  return (
+    <>
+    <ShowPage
+      pageType="tokens"
```

Comment:
this needs to say 'global_props' (global_props is the default so just removing this should be enough)